www/web-proxy-useracl/src/opnsense/scripts/OPNsense/ProxyUserACL/reload.php

```diff
@@ -39,7 +39,18 @@
 
 array_map('unlink', glob("/usr/local/etc/squid/ACL_useracl_*.txt"));
 foreach ($mdlProxyUserACL->getNodeByReference('general.ACLs.ACL')->getNodes() as $acl) {
-    file_put_contents("/usr/local/etc/squid/ACL_useracl_" .
-        $acl["Priority"] . ".txt", $acl["Name"] . "\n" .
-        ($acl["Group"]["user"]["selected"] == "1" ? $acl["Name"] . "@" . $domain . "\n" : ""));
+    file_put_contents("/test.txt", $acl["Group"]);
```

Comment:
Never used this plugin, but test.txt shouldn't be around in stable code .. 
Has anyone loaded this branch and tested it?

Reply:
Indeed it shouldn't been there, it was just for debugging purposes as I wasn't being able to get the right values from the structure. Anyway, I deleted this line. @mimugmail @pkejval